packages/descriptions/src/descriptions-cell.ts

```diff
@@ -16,39 +17,73 @@ export default defineComponent({
   },
   setup(props) {
     const descriptions = inject(elDescriptionsKey, {} as IDescriptionsInject)
+    const item = computed(() => {
+      return {
+        label: props.cell?.props?.label || '',
+        span: props.cell?.props?.span || 1,
+        width: props.cell?.props?.width || '',
+        minWidth: props.cell?.props?.['min-width'] || props.cell?.props?.minWidth || '',
+        align: props.cell?.props?.align || '',
+        labelAlign: props.cell?.props?.['label-align'] || props.cell?.props?.labelAlign || '',
```

Comment:
Does Vue help us to normalize prop name?

Reply:
How to do?